docs/tutorial/django.rst

```diff
@@ -328,12 +332,12 @@ Create the file ``time_app/urls.py`` with the following contents:
 .. literalinclude:: code/django/time_app_urls.py
     :language: python
 
-Open the file ``django_hello_world/urls.py`` and insert the following contents:
+Open the file ``django_hello_world/urls.py`` and replace its contents with the following:
```

Comment:
I think some of these lines are too long, you can check the linting output for the exact lines